packages/ui-kit/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,122 @@
+import { Button } from '@mui/base/Button'
+import { Popper } from '@mui/base/Popper'
+import { Input } from '@mui/base/Input'
+import { useAutocomplete } from '@mui/base/useAutocomplete'
+import * as React from 'react'
+import classnames from 'classnames'
+import componentStyles from './Autocomplete.module.scss'
+import { useCombinedRefs } from '../../helpers'
+import { AutocompleteProps } from './Autocomplete.types'
+
+const ChevronUpIcon = () => (
```

Comment:
Shall we move those icons to a new separate `components/Icons.tsx` component?

Reply:
Since those icons will be shared between the autocomplete and the select I think we can refactor this way